site/app/controllers/MiscController.php

```diff
@@ -40,9 +40,15 @@ private function checkValidAccess($is_zip, &$error_string) {
         }
         // from this point on, is not a zip
         // do path and permissions checking
+
         $dir = $_REQUEST['dir'];
         $path = $_REQUEST['path'];
 
+        if(!empty($_GET['dir']) && !empty($_GET['path'])) {
+            $dir = $_GET['dir'];
+            $path = $_GET['path'];
+        }
```

Comment:
Why is this check necessary? Are we setting `$_POST['dir']` and `$_GET['dir']` at the same time in this controller?

Reply:
It seems like we are not using $_POST to send to this funciton (wasn't sure originally since $_REQUEST was being used), display_file is only used with $_GET and therefore $_REQUEST is unnecessary so I have removed this explicit check and set both $dir and $path to their appropriate $_GET values in my most recent commit.